scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -23,19 +23,80 @@ source "$(dirname "${0}")"/../functions
 L_PWD=$(pwd)
 cd "${DEVON_IDE_HOME}" || exit 255
 
+# Install Eclipse plugins by feature group id
 # $1: plugin feature group ID
 # $2: plugin URL (update site)
 function doInstallEclipsePlugin() {
-  local plugin="${1/\.feature\.group*/}"
-  echo "Installing eclipse plugin ${plugin} from ${2}"
-  # shellcheck disable=SC2086
-  "${ECLIPSE_HOME}/eclipse" -nosplash -vm "${ECLIPSE_JAVA_HOME}/bin/java" -application org.eclipse.equinox.p2.director -repository "${2}" -installIU "${1}" -vmargs ${ECLIPSE_VMARGS}
-  result=${?}
-  if [ "${result}" != 0 ]
+  # Remove feature.group from each plugin id
+  local plugins="${1//.feature.group/}"
+  echo "Installing Eclipse plugin '${plugin_name}' from URL ${2}"
+
+  if [ -z "${plugins}" ]
   then
-    doEchoAttention "Installation of plugin ${plugin} failed. Please try to install manually or rerun setup (devon eclipse setup)."
+    doEcho "No plugins to install."
+    return
   fi
```

Comment:
Did this really happen for you?
Anyhow, fair to add this check so feel free to set this to resolved immediately.

Reply:
This will most likely never happen unless the .properties file has an empty plugin_id parameter. Nevertheless, I think It's not a bad idea to check the edge cases